### ISSUE TITLE
Links for info about branches

### ISSUE THREAD

**Issue by alanzirpoli** [DotBow/Blender-Launcher#96]
Would be very useful a quick link on each branch to the relative dev page to look up information about it.

**Comment by DotBow** [comment]
Hi!

Could you show examples of links you are talking about?

**Comment by DotBow** [comment]
Hi!

Could you show examples of links you are talking about?

**Comment by alanzirpoli** [comment]
Sure. This for example is the related page for this branch for example. (I belive)

https://developer.blender.org/rB05f500580b6f0e5bdad5f432e1734b639b47816e
![image](https://user-images.githubusercontent.com/63505397/132985793-d3667289-b31d-4988-b435-bc8f51d7b2e8.png)

When I find this branch I don't know what would "level set nodes" might mean. So a link to that branch that could provide some information could be very useful.

**Comment by DotBow** [comment]
The link you've provided is just a _latest commit that included in the build of that branch_ and it doesn't include the actual description of branch.
I think what you are looking for is this one https://developer.blender.org/diffusion/B/browse/geometry-nodes-level-set-nodes/, however there is no any description too.

**Comment by alanzirpoli** [comment]
You are absolutely right. My hope was just to understand what's a branch is for to see how use that branch. I don't know if there's any place where that information can be found, but I've planned to ask Pablo on blender today.
I'll let you know if there's anything useful about it.
Thanks
